docs/how-we-work.md

```diff
@@ -72,6 +72,16 @@ Major, cross-cutting refactoring efforts fit within this category. Our goals wit
 
 To introduce brand-new functionality into the package, follow this guide.
 
+##### On using RFCs
+
+We use a lightweight RFC (request for comments) process to ensure that folks have an opportunity to weigh in on design, alternatives, drawbacks, questions, and concerns. It provides a quick and easily scannable summary of what was discussed and decided.
+
+The goal is to suss out important considerations and valuable ideas as early as possible and encourage more holistic / bigger picture thinking. The goal is NOT to flesh out the perfect design or come to complete consensus before we start building.
+
+Development work on the feature may start at any point once the RFC pull request has been opened with a description of the feature. The RFC is merged once the feature work is merged.
```

Comment:
> The RFC is merged once the feature work is merged.

This will be interesting to try out. I kinda liked merging RFCs early to symbolize that the team mostly agrees on the direction.

Reply:
oh, I missed this.  Interesting.  I'm not opposed to it, but do you have any tips on remembering to merge RFCs? I have a feeling I'm going to forget.

Reply:
> I kinda liked merging RFCs early to symbolize that the team mostly agrees on the direction.

This is one of the big problems we were seeing with the existing process, though. Nobody wants to be the one who says "that's it, the design is 'done', let's get started building," so they languish. Also, there's a strong feedback loop between design and implementation, especially when it comes to UX concerns. Lots of things will look good in markdown but "feel" wrong in practice, so the design evolves naturally as we build it, and we spend less time on things we don't build.

We were hoping to alleviate this by leaving RFCs at a bit higher level of abstraction, but it's challenging to separate which points are "details we can work out later" versus "intrinsic properties we should agree on up front."

Reply:
> do you have any tips on remembering to merge RFCs? I have a feeling I'm going to forget.

Yeah I had this thought too. I figure the RFC PR will be listed in our project board, and that will serve as a good reminder. How does that sound @annthurium?

One benefit to keeping the RFC PR open until the end is that once development is complete we can make nice gifs and add them to blog post section of the RFC. That can be the final action item before we merge the RFC PR. 

And yeah, defining some level of RFC done-ness prior to actually shipping the feature into master I think will be super arbitrary and therefore hard to settle on. And I think there's value in letting the RFC remain alive and evolving as the development process surfaces new information and considerations.